src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    
+    <div class="stars stars--0">
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--3">
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--4">
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--5">
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
+
```

Comment:
Don't leave redundant blank lines
```suggestion
    </div>
```